pageserve/templates/index.html

```diff
@@ -76,11 +76,17 @@ <h2>Create New Post</h2>
 
     {% if sub_event %}
     <h2>Past Posts for Event {{ sub_event }}</h2>
+    <div class="filter_bar">
+        <div></div>
```

Comment:
Might be cleaner if you put the "Past Posts for Event [event]" in the filter bar too, like on the non-subevent case below.

The downside is that the formatting might be strange on smaller screens.

Reply:
Thanks, done https://github.com/knative-portability/large-events/pull/80/commits/cca265355d44bae15f6b54e5e2bdcb88a5facfd9
https://screenshot.googleplex.com/wF33frHDXJp